lib/ws_servers/api/index.js

```diff
@@ -92,20 +98,48 @@ module.exports = class APIWSServer extends WSServer {
     send(ws, ['info.version', VERSION])
     send(ws, ['info.exchanges', EXCHANGE_ADAPTERS.map(({ id }) => id)])
 
-    await PI.forEach(EXCHANGE_ADAPTERS, async ({ id }) => {
-      const { Market } = this.db
-      const markets = await Market.find([['exchange', '=', id]])
-
-      send(ws, ['info.markets', id, markets])
-    }).catch((err) => {
-      this.d('error sending markets to client: %s', err.stack)
-    })
+    for (const { id } of EXCHANGE_ADAPTERS) {
```

Comment:
this is not part of the algo orders, can you revert?

Reply:
i had to touch it for the paper trading feature, so its going to get refactored soon

Reply:
will do